src/sage/rings/finite_rings/element_base.pyx

```diff
@@ -142,6 +142,29 @@ cdef class FiniteRingElement(CommutativeRingElement):
         else:
             raise ValueError("unknown algorithm")
 
+    def to_bytes(self, byteorder="big"):
+        """
+        Return an array of bytes representing an integer.
+
+        Internally relies on the python ``int.to_bytes()`` method.
```

Comment:
Is there a syntax to link Python methods, similar to :meth:\`asdf\`? Maybe more advanced devs like @tscrim will know

Reply:
I am not sure. It might work as other {{{:meth:`class_name.method_name`}}} things, but that is more of a sphinx thing than anything sage-specific...